.gitignore

```diff
@@ -5,4 +5,4 @@
 .DS_Store
 venv/
 .pytest_cache/
-**__pycache__/
+**__pycache__/
```

Comment:
the same

Reply:
I thought that you want me to remove the line at the end of the file